frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetails.tsx

```diff
@@ -93,9 +96,11 @@ const BareMetalHostDetails: React.FC<BareMetalHostDetailsProps> = ({
             )}
             <dt>Host Addresses</dt>
             <dd>
-              Management: {getHostBMCAddress(host)}
-              <br />
-              NICs: {ips}
+              <ul className="metal3-bare-metal-host-details__host-addresses-list">
+                <li>Management: {getHostBMCAddress(host)}</li>
+                <li>NICs: {ips}</li>
+                <li>Boot MAC: {getHostBootMACAddress(host)}</li>
```

Comment:
not sure about the naming. What about `Boot Interface MAC` ? `Boot MAC address`?

Reply:
`Boot Interface MAC` sounds good

Reply:
renamed